nullaway/src/main/java/com/uber/nullaway/handlers/contract/ContractUtils.java

```diff
@@ -0,0 +1,143 @@
+package com.uber.nullaway.handlers.contract;
+
+import static com.google.errorprone.BugCheckerInfo.buildDescriptionFromChecker;
+
+import com.google.errorprone.VisitorState;
+import com.sun.source.tree.Tree;
+import com.sun.tools.javac.code.Symbol;
+import com.uber.nullaway.ErrorMessage;
+import com.uber.nullaway.NullAway;
+import java.util.Map;
+import javax.annotation.Nullable;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+
+/** An utility class for {@link ContractHandler} and {@link ContractCheckHandler}. */
+class ContractUtils {
+
+  /**
+   * Retrieve the string value inside an @Contract annotation without statically depending on the
+   * type.
+   *
+   * @param sym A method which has an @Contract annotation.
+   * @return The string value spec inside the annotation.
+   */
+  static @Nullable String getContractFromAnnotation(Symbol.MethodSymbol sym) {
```

Comment:
Should this also be called by the basic ContractHandler? Can we figure out if there is code duplication between what that handler does to parse this and this new code? (fine to open a follow up issue for that, though)

Reply:
Basically, is [this](https://github.com/uber/NullAway/pull/312/files#diff-24c260e3a0278128d4fc9b1e8926e612R227) now redundant with this method? And, again, might be worth it doing the full parsing and returning an `ImmutableList<ContractClause>` or the like 🙂